dev/com.ibm.ws.microprofile.metrics.monitor/resources/com/ibm/ws/microprofile/metrics/monitor/resources/MonitorMetrics.nlsprops

```diff
@@ -0,0 +1,27 @@
+###############################################################################
+# Copyright (c) 2018 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#CMVCPATHNAME N/A
+#COMPONENTPREFIX CWPMI
+#COMPONENTNAMEFOR CWPMI WebSphere Performance Monitoring MicroProfile Metrics API
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#ISMESSAGEFILE true
+# #########################################################################
+#-----------------------------------------------------------------------------------------------------------------------------
+# Messages used by MonitorMetrics Impl -  range is 2000-2199
+#-----------------------------------------------------------------------------------------------------------------------------
+FEATURE_REGISTERED=CWPMI2001I: Monitoring metrics can be retrieved through mpMetrics.
+FEATURE_REGISTERED.explanation=Feature monitor-1.0 and mpMetrics-1.1 are enabled and monitoring metrics can be retrieved through mpMetrics.
```

Comment:
Either change "Feature" to "Features" or use "The monitor-1.0 and mpMetrics-1.1 features are enabled..."